app/controllers/v1/content_blocks_controller.rb

```diff
@@ -20,8 +20,15 @@ def index
     end
 
     def store
-      content_path = params[:content_block][:content_path]
-      @content_block = @content_blocks_service.create_or_update(content_path, content_block_params)
+      block_params = params[:content_block]
+      content_path = block_params[:content_path]
+      predecessor_version = block_params[:version] ? block_params[:version].to_i : nil
```

Comment:
maybe it would be nice to have a method for this? or would it be enough to also convert `nil` to 0?